ethcore/src/tx_filter.rs

```diff
@@ -122,7 +124,7 @@ mod test {
 	use transaction::{Transaction, Action};
 	use tempdir::TempDir;
 
-	/// Contract code: https://gist.github.com/VladLupashevskyi/a157d8162db85de67de522eeb7c8ee85
+	/// Contract code: https://gist.github.com/VladLupashevskyi/84f18eabb1e4afadf572cf92af3e7e7f
```

Comment:
> bool - flag whether #'sender depends on #'to' and #'value'. If it's false
     *    the result will be cahed.

`s/cahed/cached`` and also it seems it's the other way around, right? If the result is `true` it will be cached?

"Caching" also seems a bit too harmless for this for me. We should emphasize that if `true` is returned the same permissions will be applied for all transactions from the same sender WITHOUT calling the contract again.

Reply:
@tomusdrw I agree the explanation was too confusing.

I updated gist where flag description is more clarified.